src/plugins/data_table/server_side_data_table.js

```diff
@@ -70,13 +70,49 @@ var ServerSideDataTable = (function() {
 
     // dataTables callback to allow addition of params to the ajax request
     _addServerParams : function(aoData) {
+      if (this.simpleParams) {
+        var sortBy, sortDir, limit, start, requestId;
+
+        var indexOfSortedColumn = this._getDataTableParam(aoData, "iSortCol_0");
```

Comment:
Wouldn't this limit the number of columns it can sort by default to 1? Even though our current Ext lib is limited to 1, DataTables can do multi-sort. Do we want to hard code this to 1 for Simple Params.